mimic/model/keypair_objects.py

```diff
@@ -0,0 +1,30 @@
+"""
+Keypair objects for mimic
+"""
+
+from characteristic import attributes
+
+@attributes(['name', 'public_key', 'user_id'])
+class KeyPair(object):
+    """
+    A KeyPair object
+    """
+    static_defaults = {
+        "fingerprint": "35:9d:d0:c3:4a:80:d3:d8:86:f1:ca:f7:df:c4:f9:d8",
+        "private_key": """-----BEGIN RSA PRIVATE KEY-----\nMIICXAIBAAKBgQC9mC3WZN9UGLxgPBpP7H5jZMc6pKwOoSgre8yun6REFktn/Kz7\n
```

Comment:
I believe we can just return a random string here in this scenario since the mimic consumers of the end point just verify the end point returns ok and that the key name persists(or public key if specified).

That private key scenario only comes up when a user requests th keypair api to create a server for you instead of specifying one:
http://docs.rackspace.com/servers/api/v2/cs-devguide/content/CreateKeyPair.html
So returning a pseudo random string here should be ok instead.

Will update this pr with that.